schedule/yast/maintenance/yast-mru-install_sle15sp3_dev.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: yast-mru-install_dev
+vars:
+  DUD_ADDONS: sdk
+  YUI_REST_API: 1
+schedule:
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
```

Comment:
we don't need gnome but we could have `default_systemd_target`
the schedule cannot be empty, we need a feature for that, otherwise the ci will complain.